CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Autoconfigure now supports an option to disable the SDK.
+  If `otel.sdk.disabled=false`, `AutoConfiguredOpenTelemetrySdk#getOpenTelemetrySdk()`
```

Comment:
Shouldn't it be `true`?
```suggestion
  If `otel.sdk.disabled=true`, `AutoConfiguredOpenTelemetrySdk#getOpenTelemetrySdk()`
```

Reply:
`otel.sdk.disabled=false` means active... which is what we want. The spec has details why it was done like this.

Reply:
will fix